sqe-database-access/TextRepository.cs

```diff
@@ -325,10 +366,243 @@ UserInfo editionUser
 			return new LineData { LineId = lineId, LineName = lineName };
 		}
 
+		/// <summary>
+		///  Inserts new line as fitstline into an existing text fragment which masu contasin
```

Comment:
There are some typos in this comment....

Reply:
Thanks.